pkg/metrics/metrics_test.go

```diff
@@ -0,0 +1,67 @@
+package metrics
+
+import (
+	"testing"
+
+	// . "github.com/codeready-toolchain/host-operator/test"
+	prom_testutil "github.com/prometheus/client_golang/prometheus/testutil"
```

Comment:
I would prefer not using the snake case (but it's just my opinion, nothing that I would request ;-)):
```suggestion
	promtestutil "github.com/prometheus/client_golang/prometheus/testutil"
```

Reply:
Updated 🙂